### PR TITLE
Prepare NEWS for release

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -5,29 +5,131 @@ NEWS
 HEAD
 ====
 
+Features
+--------
+
+- Core
+  - New simplified policy language (670_)
+  - Option to choose between a global and per endpoint connection tracking table (659_)
```
Comment (aanm) [inline NEWS.rst]: - Option to choose between a global (default) and per endpoint connection tracking table (659_)

File: NEWS.rst [hunk 2]
```diff
@@ -5,29 +5,131 @@ NEWS
...
+  - Parallel endpoint BPF program & policy builds (424_, 587_)
+  - fluentd logging integration (758_)
+  - IPv6 proxy redirection support (818_)
+  - Transparnt ingress proxy redirection (773_)
```
Comment (aanm) [inline NEWS.rst]: s/Transparnt/Transparent/

File: NEWS.rst [hunk 3]
```diff
@@ -5,29 +5,131 @@ NEWS
...
+  - Consider all labels for identity except dynamic k8s state labels (849_)
+  - Reduced size of cilium binary from 27M to 17M (554_)
+  - Add filtering support to ``cilium monitor`` (673_)
+  - Allow rule now supports matching multiple labels (638_)
+  - Separate runtime state and template directory for security reasons (537_)
+  - Ability to specify L4 destination port in policy trace (650_)
+  - Improved log readability (499_)
+  - Optimized connection tracking map updates per packet (829_)
+  - New --kvstore and --kvstore-opt flag (767_)
```
Comment (aanm) [inline NEWS.rst]: `  - New --kvstore and --kvstore-opt flag (767_) (Replaces ``--consul``, ``--etcd`` and ``--local`` flags)

File: NEWS.rst [hunk 4]
```diff
@@ -5,29 +5,131 @@ NEWS
...
+  - Configurable clang path (620_)
+  - Updated CNI to 5.2.0 (529_)
+  - Updated Golang to 1.8.3 (853_)
+  - Bump k8s client to v3.0.0-beta.0 (646_)
+- Kubernetes
+  - Support L4 filtering with v1beta1.NetworkPolicyPort (638_)
+  - ThirdPartyResources support for L3-L7 policies (795_, 814_)
+  - Per pod policy enablement based on policy selection (815_)
```
Comment (aanm) [inline NEWS.rst]: - Per pod policy enablement based on policy selection (815_) (Kubernetes network policy v1 compliance https://github.com/kubernetes/kubernetes/pull/39164)

File: NEWS.rst [hunk 5]
```diff
@@ -5,29 +5,131 @@ NEWS
...
-Features
---------
+- Core
+  - Endpoints are displayed in ascending order (474_)
+  - Warn about insufficient kernel version when starting up (505_)
+  - Work around Docker <17.05 disabling IPv6 in init namespace (544_)
+  - Only generate human readable ASM output if DEBUG is enabled (599_)
+  - Switch from package syscall to x/sys/unix (588_)
+  - Respond to ARP also when temporary drop all policy is applied. (724_)
+  - Fixed several BPF resource leakages (634_, 684_, 732_
```
Comment (aanm) [inline NEWS.rst]: missing`)`

**Review by jrajahalme — DISMISSED** [review]
Looks good top me. Not familiar with all the referenced issues though.

**Comment by borkmann** [comment]
Looks good, feel free to also add PRs for #828 #736 #764 .

**Comment by tgraf** [comment]
> Not sure if André's comments were already addressed.

Yes, I addressed all his comments

**Comment by tgraf** [comment]
> Looks good, feel free to also add PRs for #828 #736 #764 .

Added

**Pull request merged**